src/utils/index.js

```diff
@@ -17,6 +17,38 @@ exports.flatListToHierarchical = (
   return tree
 }
 
+// Create a list of doc nodes from the yaml file that fit the existing sidebarnav components.
+exports.navMenuListFromYaml = (nodes, titles) => {
+  let newNav = []
+  nodes.forEach((node) => {
+    // Section/parents with children items
+    let newItem = {
+      id: node.id,
+      title: node.section.name,
+      path: "#",
+      parent: null,
+      target: null,
+    }
+    newNav.push(newItem)
+    node.section.items.forEach((item) => {
+      // If yaml item doesn't have title, look for same document from other title sources
+      if (null === item.label) {
+        let found_item = titles.find((elem) => elem.uri === item.uri)
+        item.label = found_item ? found_item.title : null
```

Comment:
Would be an item in the wide bar without visual text to click on. That would be a weird state, but sign of something missing in the docs/markdown if none of the three sources could find the doc info.